alcs-frontend/src/app/features/application/proposal/proposal.component.html

```diff
@@ -40,3 +40,11 @@ <h5>Proposal Components - {{ application?.type?.label }}</h5>
   <app-proposal-nfu *ngIf="application && application.type.code === 'NFUP'"></app-proposal-nfu>
   <app-proposal-subd *ngIf="application && application.type.code === 'SUBD'"></app-proposal-subd>
 </section>
+<section>
+  <h5>Staff Comments and Observations</h5>
+  <app-inline-edit
```

Comment:
is there any length constraint on this field?

Reply:
There was none mentioned in any requirements